src/lib/redux/siteDataSlice.ts

```diff
@@ -25,6 +25,22 @@ export const EMPTY_SITE: SiteRecord = {
   tariffPlans: [],
 };
 
+export const EMPTY_FINANCIAL_SUMMARY: FinancialSummaryRecord = {
+  name: '',
```

Comment:
add `id: '',` to top of this since `FinancialSummaryRecord` now has ids!

![image](https://user-images.githubusercontent.com/35501399/110226681-3f80ab00-7ea6-11eb-8566-8ee386eb68a6.png)